build.psm1

```diff
@@ -1237,13 +1208,12 @@ function Start-PSBootstrap {
             $DotnetArguments = @{ Channel=$Channel; Version=$Version; NoSudo=$NoSudo }
             Install-Dotnet @DotnetArguments
         }
-        else
-        {
+        else {
             log "dotnet is already installed.  Skipping installation."
         }
 
-        # Install for Windows
-        if ($Environment.IsWindows) {
+        # Install Windows dependencies for building native binaries or creating Appx package
+        if ($Environment.IsWindows -and $Package) {
             $machinePath = [Environment]::GetEnvironmentVariable('Path', 'MACHINE')
             $newMachineEnvironmentPath = $machinePath
 
```

Comment:
@TravisEz13 and I talked offline. The `-package` parameter is required only for creating Appx package, and before this change `-package` takes effect only on Unix platforms. If we don't care about Appx, we can just run `Start-PSBootstrap`.
We agreed that @TravisEz13 will verify with @SteveL-MSFT about if we can remove Appx package script, and then update the packaging script and `Start-PSBootstrap` function.